lib/ContextChat/ContextChatProvider.php

```diff
@@ -54,45 +54,14 @@ public function handle(Event $event): void {
 		}
 
 		if ($event instanceof NewMessagesSynchronized) {
-			$account = $event->getAccount();
-			$mailbox = $event->getMailbox();
-			$messages = $event->getMessages();
-			$userId = $account->getUserId();
-			$client = $this->clientFactory->getClient($account);
-
-			$items = [];
-			foreach ($messages as $message) {
-				$imapMessage = $this->mailManager->getImapMessage($client, $account, $mailbox, $message->getUid(), true);
-
-				// Skip encrypted messages
-				if ($imapMessage->isEncrypted()) {
-					continue;
-				}
+			$this->jobList->add(SubmitContentJob::class, [
+				'userId' => $event->getAccount()->getUserId(),
+				'account' => $event->getAccount(),
+				'mailbox' => $event->getMailbox(),
+				'messages' => $event->getMessages(),
```

Comment:
I'm not sure this will work, as the job arguments will be JSON-serialized in the database as far as I know